module/library/categories/helper.js

```diff
@@ -141,6 +141,8 @@ module.exports = class LibraryCategoriesHelper {
 							createdAt: 1,
 							categories: 1,
 							metaInformation: 1,
+							author: 1,
+							text: 1,
```

Comment:
@borkarsaish65 text is not required. that they will get when they cal template/details api

Reply:
okk removed now